src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -908,6 +913,7 @@ public PreferenceSettings()
             ReadNotificationIds = new List<string>();
             DynamoPlayerFolderGroups = new List<DynamoPlayerFolderGroup>();
             backupLocation = string.Empty;
+            NotificationsDefaultTimeOut = 10000;
```

Comment:
hi @Enzo707 Based on my previous comment, would you move this to https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Configuration/Configurations.cs#L14? and not use a property on the current class?

Reply:
I dont think we need customers to set this value so a const value in internal config seems enough to me

Reply:
Sure, no problem

Reply:
Does it make sense to add `NotificationsDefaultTimeOut` in `Notifications.dll.config` so that advanced user will be able to customize the value?

Reply:
hi @avidit It may make sense to implement this for all network-related features and expose it in preferences as a whole. I do not see a strong need for our customer to customize the threshold for notifications feature alone. Once the no-network mode has been fully tested and introduced, I think it would put us in a better situation to check network related settings.